src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.dsde.rawls.graph
+
+import java.util.logging.{Logger, LogManager}
+
+import com.tinkerpop.blueprints.impls.orient.{OrientBaseGraph, OrientGraph}
+import org.scalatest.{FunSuite, BeforeAndAfter}
+
+trait OrientDbTestFixture extends FunSuite with BeforeAndAfter {
+  var graph: OrientBaseGraph = _
+  val testDbName: String
+
+  def initializeGraph
+
+  before {
+    // TODO find a better way to set the log level. Nothing else seems to work.
+    LogManager.getLogManager().reset()
```

Comment:
OrientDB uses it. If it's not disabled / set to severe, you'll get tons of messages for each test.